tools/RazorPageGenerator/Program.cs

```diff
@@ -75,43 +75,42 @@ private static void GenerateCodeFile(string cshtmlFilePath, string rootNamespace
                 generatedTagHelperContext: new GeneratedTagHelperContext());
             var engine = new RazorTemplateEngine(host);
 
-            using (var fileStream = File.OpenText(cshtmlFilePath))
+            var preProcessedCode = File.ReadAllText(cshtmlFilePath);
+            var startIndex = 0;
+            while (startIndex < preProcessedCode.Length)
```

Comment:
Could we move this into a separate method? `ProcessFileIncludes` or something like that?